nucliadb_dataset/tests/integration/test_question_answer_streaming.py

```diff
@@ -66,8 +66,6 @@ def test_question_answer_streaming(sdk: NucliaDB, qa_kb: KnowledgeBoxObj):
     assert len(loaded_array["answer_paragraphs"][1]) == 1
     assert len(loaded_array["answer_paragraphs"][2]) == 1
 
-    assert any(loaded_array["cancelled_by_user"])
```

Comment:
`loaded_array["cancelled_by_user"] = [[false, false, false]]`

The pyarrow update changes `[false, false, false]` to be falsy (it was truthy before)

Reply:
I think not because the fixture is not setting this explicitly in any form, so I don't think it makes a lot of sense.